flusher.go

```diff
@@ -41,6 +41,7 @@ func (s *Server) Flush(ctx context.Context) {
 	s.Statsd.Gauge("gc.pause_total_ns", float64(mem.PauseTotalNs), nil, 1.0)
 	s.Statsd.Gauge("mem.heap_alloc_bytes", float64(mem.HeapAlloc), nil, 1.0)
 	s.Statsd.Gauge("flush.flush_timestamp_ns", float64(flushTime), nil, 1.0)
+	s.Statsd.Count("heartbeat", 1, []string{"version_commit:" + VERSION}, 1)
```

Comment:
We should instrument this as an "uptime" metric, e.g. how many milliseconds have passed since we last emitted this metric. It is brittle to have this depend on the flush interval.

Reply:
I would also call this metric `uptime_ms` or something similar.

Reply:
Also I don't think this is an appropriate place to put something like this; this should go closer to the main function, since this has nothing to do with flushing metrics.

Reply:
Make sense! Done!